plugins/com.google.cloud.tools.eclipse.appengine.localserver/src/com/google/cloud/tools/eclipse/appengine/localserver/server/LocalAppEngineServerLaunchConfigurationDelegate.java

```diff
@@ -448,7 +448,8 @@ public void launch(ILaunchConfiguration configuration, String mode, final ILaunc
           generateServerRunConfiguration(configuration, server, mode, runnables);
       if (ILaunchManager.DEBUG_MODE.equals(mode)) {
         int debugPort = getDebugPort();
-        setupDebugTarget(devServerRunConfiguration, launch, debugPort, monitor);
+        devServerRunConfiguration =
```

Comment:
This should work though my inclination is to use a new local variablr here instead of overwriting the old one; e.g. debuggingConfiguration

Reply:
It is weird. Though since we only update it in debug mode, it would be more confusing to put an ?: check below to determine the appropriate instance. We could pass in the builder instead.